### PR TITLE
DOCSP-38478: bson ext json

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-scala/blob/master/REVIEWING.md)

**Note: this is part of a consolidation effort, so your review should
not address style guide issues or phrasing unless there are errors**

JIRA - <https://jira.mongodb.org/browse/DOCSP-38478>
Staging - <https://preview-mongodbrustagir.gatsbyjs.io/scala/DOCSP-38478-bson-ejson/bson/extended-json/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/bson/extended-json.txt [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+       collection.insertOne(Document(json)).head()
+     }
+   } catch {
+     case ex: Exception => println("Bummer, an exception happened.")
+   }
+
+The ``Document(<json>)`` companion helper method constructs an instance of a
+``JsonReader`` with the given string and returns an instance of an
+equivalent ``Document`` instance. ``JsonReader`` automatically detects the JSON
+flavor in the string, so you do not need to specify it.
```
Comment (norareidy) [inline source/bson/extended-json.txt:106]: S: looks like there's a note included in the [original document](https://mongodb.github.io/mongo-java-driver/5.0/driver-scala/bson/scala-extended-json/#reading-json) that's missing here, so I'd add that

**Review by norareidy — APPROVED** [review]
LGTM with a comment!

File: source/bson/extended-json.txt (hunk 1 above)
Comment (rustagir) [reply source/bson/extended-json.txt:106]: I can't locate the source repo for the `tools` link in that note, so I left it out.

Comment (norareidy) [reply source/bson/extended-json.txt:106]: Ah gotcha, I agree with leaving it out then!

**Pull request merged**